src/manager.js

```diff
@@ -103,14 +103,15 @@ AdManager.prototype.initGoogleTag = function() {
 */
 AdManager.prototype.initAmazonA9 = function() {
   try {
-    this.amznads = amznads;
+    amznads.apiReady;
   } catch(e) {
     if(e.name == "ReferenceError") {
       this.amznads = false;
       console.log('bulbs-public-ads-manager: amznads is not defined');
     }
   }
   if(this.amznads) {
```

Comment:
I feel like we probably no longer need the try/catch block now that we'll have the `apiReady` boolean.  That's set here by the way @collin: https://github.com/theonion/omni/pull/909/files